base/abstractarray.jl

```diff
@@ -968,6 +968,22 @@ _unsafe_ind2sub(sz, i) = (@_inline_meta; _ind2sub(sz, i))
     setindex!(A, X, inds...)
 
 Store values from array `X` within some subset of `A` as specified by `inds`.
+
+# Examples
+```jldoctest
+julia> A = zeros(2, 2, 2, 2);
```

Comment:
Use a smaller matrix such that we can show if all after `setindex!`ing? Perhaps also unnecessary to use `CartesianIndex`? Proposal:
````
```jldoctest
julia> A = zeros(2,2);

julia> setindex!(A, [10, 20], [1, 2]);

julia> A[[3, 4]] = [30, 40];

julia> A
2×2 Array{Float64,2}:
 10.0  30.0
 20.0  40.0
```
````